### PR TITLE
Final fixes to the Rust SIMD LP

### PR DESCRIPTION
**Pull request by markos**
Before submitting a pull request for a new Learning Path, please review [Create a Learning Path](https://learn.arm.com//learning-paths/cross-platform/_example-learning-path/)
- [X] I have reviewed Create a Learning Path

Please do not include any confidential information in your contribution. This includes confidential microarchitecture details and unannounced product information. No AI tool can be used to generate either content or code when creating a learning path or install guide.

- [X] I have checked my contribution for confidential information

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the Creative Commons Attribution 4.0 International License.

### CODE REVIEW

**Comment by markos** [comment]
@pareenaverma I still see it's under review and not visible in production, is there anything else left to fix?

**Comment by lizwar** [comment]
Hi @markos it's just under editorial review so should be live shortly.

**Comment by madeline-underwood** [comment]
Hi @markos, I was wondering if you could possibly provide a few clarifications:

- Where you use the words "monolithic" and "idiomatic", the sense is not that clear - could you rephrase with something a little simpler and easier to understand?
- The intent in page "Arm SUMD on Rust" could be made a bit clearer from the outset, instead of a drop by drop to find it out e.g., we will show X, Y, Z.... Also, what does "There is a recent Arm Community blog post about [Neon Intrinsics in Rust ](https://community.arm.com/arm-community-blogs/b/architectures-and-processors-blog/posts/rust-neon-intrinsics). The differences listed are expanded and explained in this Learning Path with examples." - What does this bit mean, and what are the differences?
- Can you explain "rustc nightlies" so that it is clear for a wider audience?
- The language on the Matrix transpose is a little unclear at times (e.g. "it’s just that the C compiler doesn’t care that much... well, C is not a human that cares much or less, it either enforces X or it does not ....") - could you possibly rewrite this bit?
- Where you mention DCT algorithms, what does this mean? 

Many thanks for your contribution and for helping us ensure that we make our content as clear and as accessible as possible.

**Comment by markos** [comment]
Hi @madeline-underwood, thank you for the review:
* "idiomatic Rust" is a term that is used to describe code that is written in optimally written Rust, code that takes advantage of Rust's unique features as much as possible. Same applies to other languages, but the term became popular recently with Rust. I would not change the term, but I could perhaps add this description.
* reg monolithic, I understand it can be confusing now that I am reading it again, I will remove that and replace with "is not added in an isolated manner, like in C compilers".
* regarding the intent, I think it's clear, "learn the basics of Arm SIMD programming in Rust". It's definitely not a Rust tutorial, it assumes at least some familiarity with Rust. I think one of the review suggestions was to put that notice in a box. The notice should be enough
* The differences are referring to programming using SIMD intrinsics in C and Rust. They are similar, but there are some minor differences, these differences are expanded in this LP. I think this will be obvious to the reader.
* Rust nightlies refer to "Rust compiler nightly builds", which are compiler versions targeted to developers mostly. I can expand that.
* I will rewrite the C compiler part to avoid the anthropomorphism :)
* DCT stands for Discrete Cosine Transform, it's similar to Fourier Transforms which are used in signal analysis, in this case they are a vital component of video codecs. I will add that part.

Let me know if you agree with the above changes
